Maestro/subscriptions.json

```diff
@@ -717,6 +717,36 @@
         }
       }
     },
+    // Update dependencies in CoreFX release/3.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/release/3.0/packages.semaphore",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/release/3.0.1/packages.semaphore",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/standard/release/3.0.0/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/buildtools/release/2.2/Latest.txt"
```

Comment:
2.2 is missing a **lot** for 3.0 builds. However, since buildtools needs to be gone for 3.0, we could have buildtools master track 3.0 rather than splitting now. We could branch later if *somehow* buildtools is still used for 3.1/4.0 and needs new fixes.

Reply:
I like the idea of using master for the buildtools dependency, considering we're aiming to get rid of it soon. Is the same true for Core-Setup?

Reply:
Yep, all repos need to get rid of buildtools dependencies for source-build. (And core-setup is up to date on buildtools master versions.)